app/main/views/login.py

```diff
@@ -23,6 +23,8 @@
 
 @main.route('/login', methods=["GET"])
 def render_login():
+    if current_user.is_authenticated():
+        return redirect(url_for('.dashboard'))
```

Comment:
This should probably redirect to `next_url` if it's set and starts with `/suppliers`, similar to what POST endpoint does